browser/net/brave_common_static_redirect_network_delegate_helper.cc

```diff
@@ -50,6 +50,12 @@ int OnBeforeURLRequest_CommonStaticRedirectWork(
   GURL new_url;
   int rc = OnBeforeURLRequest_CommonStaticRedirectWorkForGURL(ctx->request_url,
                                                               &new_url);
+  const GURL adblock_url(ctx->adblock_data_or_url);
```

Comment:
this shouldn't go in `CommonStaticRedirectWork` because it's not a static redirect. It should go in https://github.com/brave/brave-core/pull/10353/files#diff-0986ef9872f15c706dc15f78ee8ef549348a10918b13d126e4483261d7171853 along with the rest of adblock and presumably after cname checks @antonok-edm ?

Reply:
Yeah, this PR shouldn't have to touch that file. I think @ShivanKaul may have been confused by https://github.com/brave/brave-core/pull/10353#discussion_r721734688